tests/functional/u/unnecessary/unnecessary_dunder_call.py

```diff
@@ -1,6 +1,6 @@
 """Checks for unnecessary-dunder-call."""
 # pylint: disable=too-few-public-methods, undefined-variable, useless-object-inheritance
-# pylint: disable=missing-class-docstring, missing-function-docstring
+# pylint: disable=missing-class-docstring, missing-function-docstring, invalid-name
```

Comment:
Let's just capitalize. The others in the file maybe don't warn because they can't be inferred or something?
```suggestion
# pylint: disable=missing-class-docstring, missing-function-docstring
```